skaffold.yaml

```diff
@@ -0,0 +1,77 @@
+# Skaffold - https://skaffold.dev/
```

Comment:
I am a bit concerned with the current implementation here. If we want something that is a sample of how to use skaffold with Istio, that sounds good - we probably could use that for a lot of other tools as well (cc @therealmitchconnors).

As is though, this looks like a much more official endorsement, which would imply that we test, support, etc.

Maybe we can move to samples/cicd/skaffold, and in the future we can add mor

Reply:
Skaffold documentation recommends being placed in root, moving to samples/cicd/skaffold until more mature.